vj4/ui/templates/components/user.html

```diff
@@ -9,7 +9,7 @@
 {% if avatar %}
   <img class="small user-profile-avatar v-center" src="{{ udoc['gravatar']|default('')|gravatar_url }}" width="20" height="20">
 {% endif %}
-  <a class="user-profile-name" href="{{ reverse_url('user_detail', uid=udoc['_id']|default(0)) }}">{{ udoc['uname']|default('') }}</a>
+  <a class="user-profile-name" href="{{ reverse_url('user_detail', uid=udoc['_id']|default(0)) }}">{{ dudoc['name_alias']|default(udoc['uname'])|default('') }}</a>
```

Comment:
I guess it should be `dudoc['name_alias']|default(udoc['uname']|default(''))`, but I'm not sure.